<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
    <addaction name="separator"/>
    <addaction name="actionExporter"/>
    <addaction name="actionFlushScene"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionPleinEcran"/>
    <addaction name="actionBarreOutils"/>
   </widget>
   <widget class="QMenu" name="menuPinceau">
    <property name="title">
     <string>Pinceau</string>
    </property>
    <addaction name="actionDiametreP"/>
    <addaction name="actionCouleurP"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuPinceau"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBarre_d_outils">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Barre d'outils</string>
   </property>
  </action>
  <action name="actionD_marrer">
   <property name="text">
    <string>Démarrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionArr_ter">
   <property name="text">
    <string>Arrêter</string>
   </property>
  </action>
  <action name="actionExporter">
   <property name="text">
    <string>Exporter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="softKeyRole">
    <enum>QAction::NoSoftKey</enum>
   </property>
  </action>
  <action name="actionFlushScene">
   <property name="text">
    <string>Vider la scène</string>
   </property>
  </action>
  <action name="actionPleinEcran">
   <property name="text">
    <string>Plein écran</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionDiametre">
   <property name="text">
    <string>Diamètre</string>
   </property>
  </action>
  <action name="actionCouleur">
   <property name="text">
    <string>Couleur</string>
   </property>
  </action>
  <action name="actionDiametreP">
   <property name="text">
    <string>Diamètre</string>
   </property>
  </action>
  <action name="actionCouleurP">
   <property name="text">
    <string>Couleur</string>
   </property>
  </action>
  <action name="actionBarreOutils">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Barre d'outils</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>Client</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBarreOutils</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolBar</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionBarreOutils</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
